index.html

```diff
@@ -349,30 +195,82 @@ <h2 class="kn_1">
             <a href="">HTML</a>
           </li>
           <li>
-           <img src="images/Ellipse_3.png" alt="CSS" />
+            <img src="images/Ellipse_3.png" alt="CSS" />
             <a href="">CSS</a>
           </li>
         </ul>
       </li>
     </ul>
   </section>
-    <footer id="contact" class="contact">
-      <div class="contact-contact-container">
+  <footer id="contact" class="contact">
+    <div class="contact-contact-container">
       <div class="container-contact">
         <h2 class="first-text-contact">Contact me</h2>
-        <p class="second-text-contact">If you have an application you are interested in developing, a feature that you need built or a
-            project that
-            needs coding. I’d love to help with it </p>
-      </div>  
+        <p class="second-text-contact">If you have an application you are interested in developing, a feature that you
+          need built or a
+          project that
+          needs coding. I’d love to help with it </p>
+      </div>
       <form class="container-form" action="https://formspree.io/f/xbjwakbe" method="post">
         <input id="name" class="text-area" type="text" name="name" maxlength="30" required placeholder="name">
-        <input id="email" class="text-area" type="email" name="email"  required
-          placeholder="email">
-        <textarea id="message" class="text-area-large" placeholder="Write your message" maxlength="500" required></textarea>
+        <input id="email" class="text-area" type="email" name="email" required placeholder="email">
+        <textarea id="message" class="text-area-large" placeholder="Write your message" maxlength="500"
+          required></textarea>
         <button class="btn-container-contact" type="submit" name="submit">Get In Touch</button>
       </form>
+    </div>
+  </footer>
+  <div class="card-popup-window">
+    <div class="popup-main-container">
+      <div class="main-container">
+        <div class="title">
+          <h2 id="pop-title"></h2>
+        </div>
+        <div class="info">
+          <span></span>
+          <span><img class="list-item-btn" src="images/Counter.jpg" alt="bnt"></span>
+          <span></span>
+          <span><img class="list-item-btn" src="images/Counter.jpg" alt="bnt"></span>
+          <span></span>
+        </div>
+        <div class="img-container">
+          <img class="card-image" src="images/Snapshoot-Portfolio.png" alt="popup image">
+        </div>
+        <div class="left-container">
+          <div class="text-container">
+            <p></p>
+          </div>
+          <div class="about-links">
+            <ul class="languages" id="lang">
+
```

Comment:
- [ ] Just a little design implementation, kindly align your language buttons in your mobile view
    
![image](https://user-images.githubusercontent.com/66097371/163345704-46350692-fa6f-44de-83cf-d4b270fa3c11.png)